devlab/tests/generate_load.py

```diff
@@ -295,7 +295,7 @@ def update_filtering_file(self):
     def create_flavors(self):
         for flavor in self.config.flavors:
             fl = self.novaclient.flavors.create(**flavor)
-            if flavor.get('is_public') == False:
+            if flavor.get('is_public') is False:
```

Comment:
Ok, I see now :) `flavor` is set in config.py, and it may not have `is_public` field. So for this case we need to define what's default behavior if `is_public` is not set. I guess if `is_public` is not set, it should default to `True`, so wherever flavors from config are used, the check should be something like:

```
def is_public_flavor(flavor):
    return flavor.get('is_public', True)
```

And this method should be reused everywhere.

The other approach would be to add `is_public` field to all flavors in config.py.